Documentation/Input/HowToAddNearInteractivity.md

```diff
@@ -1,12 +1,30 @@
 # How to add near interaction in MRTK
+
 Near interactions come in the form of touches and grabs. Touch and grab events are raised as pointer events by the [PokePointer](Pointers.md#pokepointer) and [SpherePointer](Pointers.md#spherepointer), respectively.
 
-Listen for touch and grab events by implementing [`IMixedRealityPointerHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityPointerHandler) and looking at the type of pointer that triggers your event. If the pointer is a PokePointer, the interaction is a touch. If the pointer is a SpherePointer, the interaction is a near grab. For touch interaction it is also possible to configure objects to raise touch events by implementing the [`IMixedRealityTouchHandler`](xref:Microsoft.MixedReality.Toolkit.Input.IMixedRealityTouchHandler)
+There are three key steps to take in order to listen for touch and/or grab input events on a particular GameObject.
+
+1. Ensure the relevant pointer is registered in the main [MRTK Configuration Profile](../MixedRealityConfigurationGuide.md).
+1. Ensure the desired GameObject has the appropriate *NearInteraction** script component and [`Unity Collider`](https://docs.unity3d.com/ScriptReference/Collider.html).
+1. Implement the appropriate input handler interface on an attached script to the desired GameObject to listen for the touch/grab events.
```

Comment:
Nit: What are your thoughts around clarifying "appropriate input handler"? As a reader if I'm reading this sequentially, I kinda see this and think "hmm I wonder what an appropriate input handler is?" - I think this confusion is cleared up once I keep reading, but it does give me slight pause and question (i.e. as I'm reading, I hit a location where I don't fully understand, should I go elsewhere and interrupt my reading flow to clarify what this is, or should I just keep going?). 

I don't know if there's any way to specify examples (or just something like the specific input handler is described in the Grab and Touch interaction sections below)

Reply:
hmmm, probably worth while to list out all nearinteraction classes in a list and then put mention here of see sections below

Reply:
Actually just wrote in links to correct sections to simplify matter